.circleci/config.yml

```diff
@@ -52,89 +61,78 @@ commands:
             sudo cp /home/circleci/project/.circleci/docker-php-circleci.ini /usr/local/etc/php/conf.d/
             sudo apt-get update
             sudo apt-get install -y libpng-dev
-            sudo docker-php-ext-install gd
-  install_platformsh_tools:
-    description: "Install packages and tools for interacting with platform.sh API"
+            sudo docker-php-ext-install gd pcntl posix
+  hosts_keyscan:
+    description: "Keyscan for hosts that require SSH access"
     steps:
       - run:
-          name: Add extra OS and PHP extensions/config
-          command: |
-            sudo docker-php-ext-install pcntl posix
-      - run:
-          name: Keyscan Platform.sh region hostnames + GitHub for password-less access
+          name: Keyscan for hosts that require SSH access
           command: |
             mkdir -p ~/.ssh
-            ssh-keyscan -H ssh.$PLATFORM_REGION >> ~/.ssh/known_hosts
             ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+            ssh-keyscan -H ssh.$PLATFORM_REGION >> ~/.ssh/known_hosts
+  install_psh_cli:
+    description: "Install the Platform.sh CLI tool"
```

Comment:
Made this a command as it's good to have a fixed way of doing this. Used to be added to individual jobs as run steps within them, this is a little bit more DRY